jackson-databind/src/main/java/com/fasterxml/jackson/databind/ser/std/UUIDSerializer.java

```diff
@@ -59,7 +59,18 @@ public boolean isEmpty(SerializerProvider prov, UUID value)
     }
 
     @Override
-    public JsonSerializer<?> createContextual(SerializerProvider serializers, BeanProperty property) throws JsonMappingException { JsonFormat.Value format = findFormatOverrides(serializers, property, handledType()); Boolean asBinary = null; if (format != null) { JsonFormat.Shape shape = format.getShape(); if (shape == JsonFormat.Shape.BINARY) { asBinary = true; } else if (shape == JsonFormat.Shape.STRING) { asBinary = false; } // otherwise leave as `null` meaning about same as NATURAL } if ((null != this._asBinary) && (!Objects.equals(asBinary, _asBinary))) { return new UUIDSerializer(asBinary); } return this; }
+    public JsonSerializer<?> createContextual(SerializerProvider serializers, BeanProperty property) 
+    throws JsonMappingException { 
+        JsonFormat.Value format = findFormatOverrides(serializers, property, handledType());
+        Boolean asBinary = null; 
+        if (format != null) { 
```

Comment:
Add indentation and fix formatting but needs more review

Reply:
What more review is needed? Did someone compare this with `main`?

Reply:
The `throws` in the rest of the code have an indentation, by that logic lines 63 also should have indentation